app/src/main/java/swati4star/createpdf/util/TextToPdfAsync.java

```diff
@@ -56,4 +56,4 @@ protected void onPostExecute(Object o) {
         mOnPDFCreatedInterface.onPDFCreated(mSuccess);
     }
 
-}
+}
```

Comment:
remove this file from commit.
there is no change here

Reply:
I have deleted the file. Is this the correct way to remove it from the commit?